build-system/tasks/presubmit-checks.js

```diff
@@ -855,6 +855,7 @@ const forbiddenTermsSrcInclusive = {
       'validator/engine/validator.js',
     ],
   },
+  '\\.matches(?!_)': 'please use matches() helper in src/dom.js',
```

Comment:
`'\\.matches\\('` should suffice. Also, please capitalize "Please". 😄

Reply:
You got it!

Reply:
@choumx Sorry just double checking the matcher, if we want to pick up any use of `.matches(` wouldn't we use `\\.matches\(\`?